src/styles/fonts.module.scss

```diff
@@ -0,0 +1,303 @@
+@mixin font-body-3 {
```

Comment:
Hmm I'm not really following this numbering system. Reading the code won't give any real clue as to what CSS will be included (when using these mixins).

I think I would be more confortable with something like:
```scss
.someStyle {
  @include font-body-md
  // or
  @include font-body-2xl
}
```

Reply:
The numbering system follows the Design system. We adopted the same system in Market for these reasons:

1. For developers it's very convenient. You just click on a text in Figma and there you can see:
![image](https://github.com/user-attachments/assets/a12c4e0c-ebef-41a3-ab7f-40cca587f02b)
You immediately know that you should use `@include font-subtitle-5`. 

2. I expect that all frontends will at some point adopt the design system (we have a [repo](https://github.com/api3dao/design-tokens) syncing design tokens from Figma already). It would be easier to adopt it if we were already following the same names as the design system.

Reply:
Alright then. Ideally the design system would be a bit more descriptive.